engine/renderer/private/pipelines/ssao_pipeline.cpp

```diff
@@ -35,8 +36,8 @@ void SSAOPipeline::RecordCommands(vk::CommandBuffer commandBuffer, uint32_t curr
 {
     TracyVkZone(scene.tracyContext, commandBuffer, "SSAO Pipeline");
 
-    _pushConstants.screenWidth = _gBuffers.Size().x;
-    _pushConstants.screenHeight = _gBuffers.Size().y;
+    _pushConstants.screenWidth = _gBuffers.Size().x / 2;
```

Comment:
maybe rename the push constant to ssaoRenderTargetWidth/Height. Also rename it in the shader ofc